tiledb/platform/cert_file.h

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file   tiledb/platform/cert_file.h
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2022 TileDB, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @section DESCRIPTION
+ *
+ * Platform/machine config of the TileDB library.
+ */
+
+#ifndef TILEDB_CERT_FILE_H
+#define TILEDB_CERT_FILE_H
+
+#include <array>
+#include <filesystem>
+#include <mutex>
+#include <string>
+
+#include "os.h"
+
+namespace tiledb::platform {
+
+template <class T>
+class CertFileTraits;
+
+using PlatformCertFile = CertFileTraits<PlatformOS>;
+
+template <>
+class CertFileTraits<LinuxOS> {
+ public:
+  static const bool exists = true;
```

Comment:
Now that you point it out, I agree exists is a bit weird since there's no guarantee any of the search paths do actually exist. I'll run with enabled since I can't think of anything better.

Reply:
Done.